tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "compilerOptions": {
+    "compilerOptions": {
```

Comment:
Is this valid? Having a `compilerOptions` inside another `compilerOptions`?

Reply:
Thanks for pointing out that is redundant, I will make the correction